polyfill/spatial-navigation-polyfill.js

```diff
@@ -991,32 +970,20 @@
     offsetY = isNaN(offsetY) ? 1 : offsetY;
 
     const elementRect = getBoundingClientRect(element);
+    const hitTestPoint = {
+      middle: [(elementRect.left + elementRect.right) / 2, (elementRect.top + elementRect.bottom) / 2],
+      leftTop: [elementRect.left + offsetX, elementRect.top + offsetY],
+      leftBoottom: [elementRect.left + offsetX, elementRect.bottom - offsetY],
+      rightTop: [elementRect.right - offsetX, elementRect.top + offsetY],
+      rightBottom: [elementRect.right - offsetX, elementRect.bottom - offsetY]
+    };
 
-    const middleElem = document.elementFromPoint((elementRect.left + elementRect.right) / 2, (elementRect.top + elementRect.bottom) / 2);
-    if (element === middleElem || element.contains(middleElem)) {
-      return true;
-    }
-
-    const leftTopElem = document.elementFromPoint(elementRect.left + offsetX, elementRect.top + offsetY);
-    if (element === leftTopElem || element.contains(leftTopElem)) {
-      return true;
-    }
-
-    const leftBottomElem = document.elementFromPoint(elementRect.left + offsetX, elementRect.bottom - offsetY);
-    if (element === leftBottomElem || element.contains(leftBottomElem)) {
-      return true;
-    }
-
-    const rightTopElem = document.elementFromPoint(elementRect.right - offsetX, elementRect.top + offsetY);
-    if (element === rightTopElem || element.contains(rightTopElem)) {
-      return true;
-    }
-
-    const rightBottomElem = document.elementFromPoint(elementRect.right - offsetX, elementRect.bottom - offsetY);
-    if (element === rightBottomElem || element.contains(rightBottomElem)) {
-      return true;
+    for(const point in hitTestPoint) {
```

Comment:
Oh, I thought that the constant variable cannot work in `for .. in` loop, but it works! Interesting to know that.

Reply:
It may because 'let' and 'const' variables are created in each block("{}").